nextflow.config

```diff
@@ -416,12 +416,14 @@ params {
     }
 
     mlst {
-        singularity = "https://depot.galaxyproject.org/singularity/mlst%3A2.23.0--hdfd78af_1"
-        docker = "quay.io/biocontainers/mlst:2.23.0--hdfd78af_1"
+        version = "2.23.0"
+        singularity = "https://depot.galaxyproject.org/singularity/mlst%3A${version}--hdfd78af_1"
+        docker = "quay.io/biocontainers/mlst:${version}--hdfd78af_1"
         args = ""
         tsv_ext = ".tsv"
         json_ext = ".json"
         report_tag = "SevenGeneMLSTReport"
+        name_version = "mlst ${version}"
```

Comment:
Ideally I think it would be good to get the software version from running the software (which should be part of the channel tracking versions and writing them to a YAML file at the end of the pipeline). But, that also sounds complicated to parse out.

Right now, what you have is the simplest solution and makes the most sense. I just wanted to make a note here so we have this recorded if we ever wanted to revisit later.